src/schemas/openapi/OpenAPIDto.yaml

```diff
@@ -0,0 +1,14 @@
+type: object
+properties:
+  source:
+    oneOf:
+      - type: string
+      - type: object
+      - type: array
+        items:
+          type: string
+          format: binary
+  load:
+    type: boolean
```

Comment:
What is the source going to be? is it going to be the entire OpenAPI doc? and in what format do you want to send it?

Reply:
The type string is related to an url where the openAPI doc is located for instance:https://petstore.swagger.io/v2/swagger.json
The type object is the json content but as javascript object
The type array of binary is the file that contains the doc ( yaml or json )